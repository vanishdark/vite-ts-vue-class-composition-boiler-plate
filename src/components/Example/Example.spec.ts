import { mount } from '@vue/test-utils';
import { wrap } from 'module';
import ExampleComponent from './Example.component.vue';

test('mount component', () => {
  expect(ExampleComponent).toBeTruthy();
  const wrapper = mount(ExampleComponent, {
    props: {
      name: 'Example Component',
    },
  });
  expect(wrapper.text()).toContain('Example Component');
  expect(wrapper.html()).toMatchSnapshot();
});
