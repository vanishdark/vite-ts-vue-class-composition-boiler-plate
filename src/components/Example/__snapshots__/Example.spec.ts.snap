// Vitest Snapshot v1

exports[`mount component 1`] = `
"<div name=\\"Example Component\\" data-v-97b3bdbc=\\"\\">
  <h1 data-v-97b3bdbc=\\"\\">Example Component</h1>
</div>"
`;
